runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientConnectionPool.java

```diff
@@ -1446,10 +1526,18 @@ private void cleanupBudgetCreditorIfNecessary()
         }
 
         private void onStreamClosed(
+            long traceId,
+            long authorization,
             long streamId)
         {
+            responseRsts.add(streamId);
             streams.remove(streamId);
-            streamsByInitialIds.remove(streamId);
+
+            KafkaClientStream stream = streamsByInitialIds.get(streamId);
+            if (stream.replyAck == stream.replySeq)
```

Comment:
Should we not always set `stream.replyAck = stream.replySeq` so that a partial stream cannot block progress of connection window?